tokio-trace/src/lib.rs

```diff
@@ -27,9 +27,9 @@
 //! The core of `tokio-trace`'s API is composed of `Event`s, `Span`s, and
 //! `Subscriber`s. We'll cover these in turn.
```

Comment:
nit: maybe make the order here the same as the order they're explained in below?